### PR TITLE
Mention specifc operator in arith overflow

### PR DESCRIPTION
**Pull request by rocky**
In support of #668

### CODE REVIEW

**Comment by norhh** [comment]
@rocky , some of the tests will just check whether output matches or not between the files in  `tests/testdata/outputs_current/*` and `tests/testdata/outputs_expected/*` , so for now just copy the files in the outputs_current to the others.(I think there was some issue which is a superset of what you mentioned but that can take some time, so we will create a new issue which deals with this specific aspect).

**Comment by birdofpreyru** [comment]
@rocky well, this very failure happened in unit/functional tests included into mythril-classic codebase itself, so @norhh and @JoranHonig are your ppl to chat about it :)

In general, on one hand you are right, it is annoying when small changes in description demand updating tests; on other hand this forces you to double-check that messages were updated as intended. I believe, double-checking is good, we should just ensure that updating of tests is easy, and it is documented how to do it.

File: signatures.json [hunk 1]
```diff
@@ -1 +1 @@
...
...derLimit(uint256)"], "0x9d4468ff": ["today())"], "0x19045a25": ["recover(bytes32,bytes)"], "0xe92dfb23": ["LimitBalance(uint256)"], "0xd73dd623": ["increaseApproval(address,uint256)"], "0x66188463": ["decreaseApproval(address,uint256)"], "0xabaf5880": ["Crowdsale(uint256,uint256,uint256,address)"], "0xec8ac4d8": ["buyTokens(address)"], "0x9d735286": ["forwardFunds()"], "0x605120cf": ["validPurchase())"], "0x6e42787f": ["hasEnded())"], "0xe5c46944": ["MultiSigWallet(address[],uint256)"], "0x7065cb48": ["addOwner(address)"], "0x173825d9": ["removeOwner(address)"], "0xe20056e6": ["replaceOwner(address,address)"], "0xba51a6df": ["changeRequirement(uint256)"], "0xc6427474": ["submitTransaction(address,uint256,bytes)"], "0xc01a8c84": ["confirmTransaction(uint256)"], "0x20ea8d86": ["revokeConfirmation(uint256)"], "0xee22610b": ["executeTransaction(uint256)"], "0x784547a7": ["isConfirmed(uint256)"], "0xec096f8d": ["addTransaction(address,uint256,bytes)"], "0x8b51d13f": ["getConfirmationCount(uint256)"], "0x54741525": ["getTransactionCount(bool,bool)"], "0xa0e67e2b": ["getOwners()"], "0xb5dc40c3": ["getConfirmations(uint256)"], "0xa8abe69a": ["getTransactionIds(uint256,uint256,bool,bool)"]}
```
[the remark this answers is not in the document]
Comment (norhh) [reply signatures.json]: I think thats how the file should be from the start, there can be multiple function names for a signature, as signature is some 4 bytes or something so there is a possibility for collisions, currently I think signatures handles the files containing both list and a string.

[the remark this answers is not in the document]
Comment (rocky) [reply signatures.json]: This looks like it needs a separate issue and resolution which is really outside of this pull request.

**Comment by JoranHonig** [comment]
When the tests are done I'll merge it

**Pull request merged**